### PR TITLE
Sdk2 update

### PR DESCRIPTION
**Pull request by goelakash**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: reinforcement_learning/bandits_recsys_movielens_testbed/bandits_movielens_testbed.ipynb [hunk 1]
```diff
@@ -54,6 +54,16 @@
...
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Install sagemaker SDK v2\n",
+    "!pip install sagemaker==2.16.1"
```
Comment (ngluna) [inline reinforcement_learning/bandits_recsys_movielens_testbed/bandits_movielens_testbed.ipynb:64]: Need to check if we need this anymore. I thought the Python SDK v2 was the new default for the notebooks.

**Review by ngluna — COMMENTED** [review]
Looks good. The only question is whether or not we need to explicitly install the Python SDK v.2 . Re: I thought this was the new default?

**Comment by goelakash** [comment]
@ngluna We're still not sure whats the status of SKD v2 upgrade for all notebooks, so playing it safe by selectively updating the notebooks that our team owns (i.e., RL) first.

**Comment by ngluna** [comment]
Looks like they have. You may need to shut down your notebook instance and then restart it, but if you import sagemaker it'll work. So remove the pip install.

**Comment by ngluna** [comment]
I see it was merged. Unmerge, remove the pip install, then re-merge

**Pull request merged**